models/staging/shared/dictionary/stg_dictionary_ers_action.sql

```
@@ -0,0 +1,11 @@
+{{
+    config(
+        materialized = 'table',
+        tags=['ers']
+        )
+    }}
+select
+    code,
+    meaning,
+    display
+from {{ ref('raw_dictionary_ers_action') }}
```

SQLFluff (4.2.2)

[error] Files must end with a single trailing newline.

(LT12)